Scientific_workflows/DEAWaterbodies/DEAWaterbodiesToolkit/WOfSImputation.ipynb

```diff
@@ -0,0 +1,1810 @@
+{
```

Comment:
<p>I can't 100% tell if you prefer per-pixel averaging here or mean filling. I'm guessing the first, but can you please make it a touch more explicit in the conclusion? Maybe reproduce the good picture here and if you're going to say 'Patterns are very clearly pulled out' can you please say something like 'In the imputed time series seasonal patterns of filling and emptying are seen, whereas there is a big line of nothing in the unimputed series'. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoscienceAustralia/dea-notebooks/pull/713/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='71'/>

Reply:
<p>Clarified this and fixed the wording.</p>